app/src/main/java/org/wikipedia/views/CustomDatePicker.kt

```diff
@@ -91,10 +91,10 @@ class CustomDatePicker : DialogFragment() {
 
             init {
                 binding.root.setOnClickListener {
-                    selectedDay[Calendar.DATE] = adapterPosition + 1
```

Comment:
The getter for `adapterPosition` is deprecated, but is equivalent to `bindingAdapterPosition`.